src/components/Radio/Radio.stories.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { Radio } from "./Radio";
 
 export default {
-  title: "COMPONENTS/Radio",
+  title: "Components/Radio",
```

Comment:
```suggestion
  title: "COMPONENTS/Radio/Radio",
```
"components" needs to stay capitalized

Reply:
Just curious, why is that?